tools/spelling/spelling_dictionary.txt

```diff
@@ -764,6 +765,8 @@ intra
 ints
 invariance
 invoker
+iov
+iovcnt
```

Comment:
im wondering if these (and `rfield` below) are necessary - usually you can avoid adding to the dictionary by making them literals - but im not seeing where they are used

Reply:
the `iov` and `iovcnt` are used for proxy filter unittest there are lot of mock using it https://github.com/envoyproxy/envoy/pull/17395/files/5cdfe2b7146828ac57f9332a2432ccd1a1bbed17#diff-5346d6d1fd1c235f093f0b9800e0b2636b74a0999d11f39078578a82c217f262R1483


the `rfield` is used for https://github.com/envoyproxy/envoy/pull/17395/files/5cdfe2b7146828ac57f9332a2432ccd1a1bbed17#diff-790198a24d6c4cc17c07e52caed0fceeb7070bce698babb514412555ff78255fR567

But that isn't a line I changed, I don't know why the spell check script complains about it

Reply:
yep - these dont seem to be spellcheckable

Reply:
sorry, `iov` and `iovcnt` are used for http inspector test.

```
~/go/src/github/envoyproxy/envoy$ ./tools/spelling/check_spelling_pedantic.py check
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:437:      .WillOnce(Invoke([&data](os_fd_t fd, const iovec* iov, int iovcnt) -> Api::SysCallSizeResult {
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:438:        ASSERT(iov->iov_len >= data.size());
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:439:        memcpy(iov->iov_base, data.data(), data.size());
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:524:            Invoke([&data, ctr, copy_len, num_loops](os_fd_t fd, const iovec* iov,
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:525:                                                     int iovcnt) -> Api::SysCallSizeResult {
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:526:              ASSERT(iov->iov_len >= 1);
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:527:              memcpy(iov->iov_base, data.data() + *ctr, 1);
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:567:  absl::string_view request = "GET /index HTTP/1.0\rfield: ";
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:580:              [&data, i](os_fd_t fd, const iovec* iov, int iovcnt) -> Api::SysCallSizeResult {
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:581:                ASSERT(iov->iov_len >= 20);
./test/extensions/filters/listener/http_inspector/http_inspector_test.cc:582:                memcpy(iov->iov_base, data.data() + i, 1);
Checked 3951 file(s) and 103823 comment(s), found 19 error(s).
ERROR: spell check failed. Run 'tools/spelling/check_spelling_pedantic.py fix and/or add new words to tools/spelling/spelling_dictionary.txt'
```

Reply:
same - this is code and shouldnt be checked i think

the spellcheck needs some work i think - so if its necessary - please ignore my feedback (ill look at further - but the spellcheck code is a bit of a minefield so my attempts so far to improve have not gotten so far)

Reply:
thanks!